protos/proto/aptos/transaction/v1/transaction.proto

```diff
@@ -51,6 +51,7 @@ message Transaction {
     TRANSACTION_TYPE_STATE_CHECKPOINT = 3;
     TRANSACTION_TYPE_USER = 4;
     TRANSACTION_TYPE_VALIDATOR = 20;
+    TRANSACTION_TYPE_BLOCK_EPILOGUE = 23;
```

Comment:
This can actually be 5 or 21, right? :trollface:

Reply:
maybe 21 so it's always increasing at least :trollface:

Reply:
is 20 intentional or a mistake?

i.e. is there a reason txn_data below starts from 7 and doesn't overlap with this?

Reply:
I think it's a mistake, but @bowenyang007  @CapCap ?

Reply:
seems like original review for validator transactions was - dunno - https://github.com/aptos-labs/aptos-core/pull/11009/files/a453bea41db4e7f4a7f41d5cdc23257fc6d1e550#r1419756557

seems like txn_data below was deduping with other fields, not this enum, but that got mixed up - https://github.com/aptos-labs/aptos-core/pull/6269/files#diff-415e8460829f8a6e9e8c570675bd2c425e94b988a022f5914da912cb21141a2a


I'll update and make a comment

Reply:
It's fine either way. we just need these to be unique. 

@larry-aptos can you help test this?